### ISSUE TITLE
1.20.5: SWEEPING_EDGE Enchantment seems to be broken

### ISSUE THREAD

**Issue by aerulion** [PaperSpigot/Paper#10614]
### Expected behavior

Being able to put SWEEPING_EDGE on a Sword / Item.

### Observed/Actual behavior

Other enchants seem to work just fine, but when applying SWEEPING_EDGE, it will result in an error: https://pastebin.com/raw/J6kTz7wL

### Steps/models to reproduce

Something like
```
  @EventHandler(priority = EventPriority.MONITOR)
  public void listen(final @NotNull PlayerJumpEvent event) {
    final @NotNull ItemStack itemStack = new ItemStack(Material.DIAMOND_SWORD);
    itemStack.editMeta(itemMeta -> itemMeta.addEnchant(event.getPlayer().isSneaking() ? Enchantment.SHARPNESS : Enchantment.SWEEPING_EDGE, 1, false));
    event.getPlayer().getInventory().addItem(itemStack);
  }
```

### Plugin and Datapack List

Just a single test plugin.

### Paper version

[01:54:05 INFO]: This server is running Paper version git-Paper-19 (MC: 1.20.5) (Implementing API version 1.20.5-R0.1-SNAPSHOT) (Git: 05308fc)
You are running the latest version
Previous version: git-Paper-15 (MC: 1.20.5)

**Comment by aerulion** [comment]
I just noticed I used paper-api 1.20.4, hence the DAMAGE_ALL enchant, but the issue still occurs on a 1.20.5 plugin using userdev, even specifying the `paperweight.reobfArtifactConfiguration = ReobfArtifactConfiguration.MOJANG_PRODUCTION` property.

**Comment by Machine-Maker** [comment]
Fixed in f9397e1
